### DISCUSSION TITLE
Ventiler les achats sur plusieurs cantines dans "Mes achats"

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1259] [Idées]
**Problème :** à ce jour, dans l'outil Mes Achats, il n'est pas possible pour une cuisine de ventiler ses achats sur plusieurs cantines. Or c'est précisément ce dont auraient besoin des gestionnaires de cuisines centrales pour qu'ils se mettent à saisir leurs achats sur Mes Achats. Tout ce qu'ils peuvent faire pour l'instant, c'est saisir les achats pour la cuisine centrale sans les répartir sur les cantines satellites.

**Solution proposée :**
1. donner la possibilité à un gestionnaire de sélectionner plusieurs cantines dans le menu déroulant (cf. PJ)
2. proposer au gestionnaire de sélectionner le type de répartition du montant de la facture sur les cantines satellites concernées : une répartition équitable, ou bien sa répartition habituelle (intitulée "Ma répartition" et renseignée au moment de la création de la fiche de la cuisine centrale?), ou alors une nouvelle répartition (un champ par cantine satellite sélectionnée apparaît, dans lequel il peut renseigner un pourcentage). Cf. PJ.

<img width="1440" alt="Mes Achats 1" src="https://user-images.githubusercontent.com/93709421/161957885-8045babf-041e-4bbc-bf0e-8e72b2b77027.png">
<img width="1440" alt="Mes Achats 2" src="https://user-images.githubusercontent.com/93709421/161957895-94125b87-d736-4dd8-8442-93d6937ebfc5.png">
<img width="1440" alt="Mes Achats 3" src="https://user-images.githubusercontent.com/93709421/161957899-92d41aec-19b5-4ac3-8591-d97ce63e27cc.png">

@JenniferStephan, @alemangui & @hfroot, qu'en pensez-vous?

**Comment by hfroot** [comment]
Est-ce que il y a des cuisines centrales qui n'ont pas un système numérique pour suivre leur achats ?

**Reply by ghost** [reply]
@hfroot, étonnamment, de ce que m'on dit certains gestionnaires de cantine, ils n'ont pas d'outil de ventilation de leurs achats sur leurs clients (ex. : la petite SRC La Table de Cana). Or je pense que nous aurions intérêt à ce que les gestionnaires de cantine utilisent de plus en plus "Mes Achats".
